.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug Report
+about: Found a bug? Create a report to help us improve!
+title: "[BUG] Brief Description Here"
+labels: 'status: needs triage, type: bug'
+assignees: ''
+
+---
+
+## Description
+
+A clear and concise description of what the bug is.
+
+## Reproduction Steps
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+## What You Expected to Happen
+
+A clear and concise description of what you expected to happen.
+
+## What Actually Happened
+
+A clear and concise description of what actually happened.
+
+## Screenshots
```

Comment:
Having the logging output would also be very helpful.

There already is such a template in `docs/ISSUE_TEMPLATE/bug_report.md`.

Reply:
Ah, I think they have to live under `.github/issues/`. I'll mention the logs in there.